<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1630</width>
    <height>1258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1631</width>
      <height>1241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1606</width>
          <height>1237</height>
         </rect>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>801</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Lucida Fax&quot;;
background-color: rgb(181, 206, 227);</string>
         </property>
         <property name="title">
          <string>Select Location</string>
         </property>
         <widget class="QComboBox" name="stateBox">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>50</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>State</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>120</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>City</string>
          </property>
         </widget>
         <widget class="QListWidget" name="cityList">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>90</y>
            <width>256</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>50</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Zip</string>
          </property>
         </widget>
         <widget class="QListWidget" name="ZipList">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>50</y>
            <width>256</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="CategoryGroup">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>230</y>
           <width>1611</width>
           <height>371</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Lucida Fax&quot;;
background-color: rgb(181, 206, 227);</string>
         </property>
         <property name="title">
          <string>Categories</string>
         </property>
         <widget class="QListWidget" name="CategoryList">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>256</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="businessTable">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>30</y>
            <width>1011</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
         <widget class="QPushButton" name="FilterButton">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>150</y>
            <width>141</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Apply Filter</string>
          </property>
         </widget>
         <widget class="QPushButton" name="RemoveFilterButton">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>280</y>
            <width>141</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Remove Filter</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>800</x>
           <y>0</y>
           <width>811</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Lucida Fax&quot;;
background-color: rgb(181, 206, 227);</string>
         </property>
         <property name="title">
          <string>Zipcode Statistics</string>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string># Businesses</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="IncomeTextEdit">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>180</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>120</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Total Population</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>180</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Average Income</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="PopulationText">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>120</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="NumBusinessText">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>60</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTableWidget" name="CategoryTable">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>40</y>
            <width>281</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </widget>
        <widget class="QGroupBox" name="CategoryGroup_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>600</y>
           <width>1611</width>
           <height>381</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Lucida Fax&quot;;
background-color: rgb(181, 206, 227);</string>
         </property>
         <property name="title">
          <string>Popular/Successful Businesses</string>
         </property>
         <widget class="QTableWidget" name="PopularTable">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>20</y>
            <width>411</width>
            <height>351</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
         <widget class="QPushButton" name="PopButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>141</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Activate</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="SuccessfulBUsinesses">
          <property name="geometry">
           <rect>
            <x>1120</x>
            <y>20</y>
            <width>411</width>
            <height>351</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>224</x>
            <y>60</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Popular Businesses</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>890</x>
            <y>60</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Successful Businesses</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>1310</y>
           <width>160</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1630</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
